app/UpdateStore.tsx

```diff
@@ -7,24 +7,22 @@ import { PageHeader } from "./ui/PageHeader";
 import { FormLabel } from "./ui/FormLabel";
 import { FormGroup } from "./ui/FormGroup";
 import { FormControl } from "./ui/FormControl";
+import { Store } from "./types";
 
 export function UpdateStore(props: {
   onUpdateStoreSuccess: () => void;
   onBackClick: () => void;
-  //   prefilledDisplayName: string;
-  //   prefilledFullName: string;
-  //   prefilledSuburb: string;
+  storeDetails?: Store;
```

Comment:
in general words like `details`, `info` and `data` don't really provide any additional indication over just calling this `store`

If this is optional that means that this `UpdateStore` thing can work without it - is that true?

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/20/commits/4ee2e60b026782eb10cd1d20f21b43c3108461d3 
you'd never be able to access `UpdateStore` without being given `store` so the ? has been removed now